DD/Views/incubadoras.cshtml

```diff
@@ -1,6 +1,58 @@
 ﻿@using Umbraco.Cms.Web.Common.PublishedModels;
 @inherits Umbraco.Cms.Web.Common.Views.UmbracoViewPage<ContentModels.Incubadoras>
 @using ContentModels = Umbraco.Cms.Web.Common.PublishedModels;
+@using Umbraco.Cms.Core.Models.Blocks;
 @{
-	Layout = null;
-}
\ No newline at end of file
+	Layout = "Main.cshtml";
+    var newsHeadline = Model.EditText;
+    var fontSizeHeadline = Model.FontSizeEditor;
+    var colorHeadline = Model.SelectColor;
+    var hideHeadline = Model.Hide;
+}
+@*navigationBar*@
+<div>
+    @await Html.PartialAsync("Partials/homePage/_navigation.cshtml")
+</div>
+@*Incubadora headline Title*@
+<div class="headlineTitle">
+    @if(newsHeadline != null && hideHeadline != true)
+    {
+    <h1 id="headline" style="color:@colorHeadline">@newsHeadline</h1>
+    }
+    <script>
+        document.addEventListener("DOMContentLoaded", function () {
+            var fontSizeHeadline = @fontSizeHeadline;
+            document.getElementById("headline").style.fontSize = fontSizeHeadline + "rem";
+        });
+    </script>
+</div>
+@*Content*@
+<div class="m-5" style="color:white;">
+    @foreach (var item in Model.IncubadoraGeneralContent)
+    {
+    @*if (item.Content is ContentModels.RichTextAndImageItems richTextAndImageItems)
+        {
+            <div class="mt-3">
+                @Html.Partial("Partials/_textAndImage", richTextAndImageItems)
+            </div>
+        }*@
+        if(item.Content is ContentModels.RichTextItems text)
+        {
+    <div class="mt-3">
+        @Html.Raw(text.RichText)
+    </div>
+        }
+        if(item.Content is ContentModels.ImageCarouselRow imageRow)
+        {
+            string uniqueCarouselId = $"imageCarousel-{item.GetHashCode()}";
+            <div class="mt-3">
+                @Html.Partial("Partials/homePage/_imageCarousel", imageRow.ImageCarouselRowItems, new ViewDataDictionary(ViewContext.ViewData) { { "CarouselId", uniqueCarouselId } })
+            </div>
+        }
+    }
+</div>
+@*footer*@
+<div>
+    @*@await Html.PartialAsync("Partials/homePage/_footerHeadline")*@
+    @*@await Html.PartialAsync("Partials/homePage/_footer.cshtml")*@
```

Comment:
Avoid Commented Code

Reply:
I removed commented code like the first review request